python/ql/src/experimental/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -220,15 +220,153 @@ predicate jumpStep(Node pred, Node succ) {
 // Field flow
 //--------
 /**
- * Holds if data can flow from `node1` to `node2` via an assignment to
+ * Holds if data can flow from `nodeFrom` to `nodeTo` via an assignment to
  * content `c`.
  */
-predicate storeStep(Node node1, Content c, Node node2) { none() }
+predicate storeStep(Node nodeFrom, Content c, Node nodeTo) {
```

Comment:
At some point you probably also want to implement the `clearsContent` predicate, for example `dict.clear()` will clear dictionary contents. However, that is easiest implemented if you have use-use flow.

Reply:
Yes, it is still outstanding if we should make an effort to switch to use-use flow.